web/src/settings_invites.ts

```diff
@@ -325,6 +374,108 @@ export function on_load_success(
 
         $(".dialog_submit_button").attr("data-invite-id", invite_id);
     });
+
+    $(".admin_invites_table").on("click", ".edit-invite", function (this: HTMLElement, e) {
+        // This click event must not get propagated to parent container otherwise the modal
+        // will not show up because of a call to `close_active` in `settings.js`.
+        e.preventDefault();
+        e.stopPropagation();
+
+        const invite_id = $(this).closest("tr").attr("data-invite-id")!;
+
+        $(".dialog_submit_button").closest("tr").attr("data-invite-id", invite_id);
+        const invite: Invite | undefined = meta.invites.find(
```

Comment:
I don't think `invite` can be `undefined` since ID will always be of a valid invite.